frontend/pages/index.tsx

```diff
@@ -168,32 +170,35 @@ export default function Marketplace() {
             <Button component={Link} href='/entry/new' variant='contained'>
               Create
             </Button>
-            <FormControl
-              sx={{
-                display: 'flex',
-                alignItems: 'center',
-                maxWidth: '400px',
-                marginBottom: 3,
-              }}
-              variant='filled'
-              onSubmit={onFilterSubmit}
-            >
-              <InputLabel htmlFor='entry-filter-input'>Search</InputLabel>
-              <FilledInput
-                sx={{ flex: 1, backgroundColor: theme.palette.background.paper, borderRadius: 2 }}
-                id='entry-filter-input'
-                value={filter}
-                disableUnderline
-                onChange={handleFilterChange}
-                endAdornment={
-                  <InputAdornment position='end'>
-                    <IconButton color='secondary' type='submit' sx={{ p: '10px' }} aria-label='filter'>
-                      <SearchIcon />
-                    </IconButton>
-                  </InputAdornment>
-                }
-              />
-            </FormControl>
+            <Stack direction={'row'} spacing={0.5}>
+              <FormControl
+                sx={{
+                  display: 'flex',
+                  alignItems: 'center',
+                  maxWidth: '400px',
+                  marginBottom: 3,
+                }}
+                variant='filled'
+                onSubmit={onFilterSubmit}
+              >
+                <InputLabel htmlFor='entry-filter-input'>Search</InputLabel>
+                <FilledInput
+                  sx={{ flex: 1, backgroundColor: theme.palette.background.paper, borderRadius: 2 }}
+                  id='entry-filter-input'
+                  value={filter}
+                  disableUnderline
+                  onChange={handleFilterChange}
+                  endAdornment={
+                    <InputAdornment position='end'>
+                      <IconButton color='secondary' type='submit' sx={{ p: '10px' }} aria-label='filter'>
```

Comment:
In cases where we're using such a smaller amount of padding/margin it's better to use the theme spacing if possible. `p:3` for example. Although due to base value probably not being divisible by 10 you might not achieve the same padding - so if it _needs_ to be exactly `10px` then ignore this comment